.github/workflows/lint.yaml

```diff
@@ -2,7 +2,7 @@ name: HOPR Lint and fix
 
 on:
   push:
-    branches: ['**']
+    branches: ['**', '!debug/**']
```

Comment:
why exclude linting in debug?

Reply:
This is meant to be used to speed up developing workflow, not meant to be a final change, see WIP